worlds/ff6wc/WorldsCollide/args/objectives.py

```diff
@@ -90,13 +99,28 @@ def __init__(self, letter, result, conditions, conditions_required_min, conditio
                 condition_args = values[value_index : value_index + condition_arg_count]
                 value_index += condition_arg_count
 
+                # if there are no condition args at all, this is an error
+                # this is something like -oa 2.1.1.3 where there needs to be something after 3 (either r or a number)
+                if condition_args == []:
```

Comment:
a better way to check for empty
```suggestion
                if len(condition_args) < 1:
```
`== []` creates a new list just to compare with it. That wastes time and memory.

Reply:
Done!